ArcGISRuntimeSDKQt_CppSamples/Layers/DisplayFeatureLayers/DisplayFeatureLayers.cpp

```diff
@@ -0,0 +1,229 @@
+
+// TRADE SECRETS: ESRI PROPRIETARY AND CONFIDENTIAL
```

Comment:
These copyrights should be removed in favor of the apache copyrights for github.com projects